sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_authentication_policy.py

```diff
@@ -0,0 +1,85 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+from typing import TYPE_CHECKING
+
+from azure.core.exceptions import ServiceRequestError
+from azure.core.pipeline.policies import AsyncHTTPPolicy
+
+from ._async_exchange_client import ACRExchangeClient
+
+if TYPE_CHECKING:
+    from azure.core.credentials_async import AsyncTokenCredential
+    from azure.core.pipeline import PipelineRequest, PipelineResponse
+    from typing import Optional, Dict, Any, Union
+
+
+def _enforce_https(request):
```

Comment:
Is there a reason this method is implemented in both the sync and async policies? Seems like extra maintenance?

Reply:
I placed this in `_helpers.py`, thank you!